modin/pandas/io.py

```diff
@@ -716,17 +716,17 @@ def to_pickle(
 @enable_logging
 def read_spss(
     path: Union[str, pathlib.Path],
-    usecols: Union[Sequence[str], type(None)] = None,
+    usecols: Sequence[str] | None = None,
     convert_categoricals: bool = True,
 ):  # noqa: PR01, RT01, D200
     """
     Load an SPSS file from the file path, returning a DataFrame.
     """
+    _, _, _, kwargs = inspect.getargvalues(inspect.currentframe())
+
```

Comment:
let's not introduce this line... it's kinda slow and not that much useful when we have only 3 arguments